src/v2i-hub/PedestrianPlugin/src/PedestrianPlugin.cpp

```diff
@@ -143,54 +168,60 @@ void PedestrianPlugin::UpdateConfigSettings()
 	// This method does NOT execute in the main thread, so variables must be protected
 	// (e.g. using std::atomic, std::mutex, etc.).
 
-	int instance;
 	std::lock_guard<mutex> lock(_cfgLock);
-
-	GetConfigValue<string>("WebServiceIP",webip);
-	GetConfigValue<uint16_t>("WebServicePort",webport);
-	GetConfigValue<string>("WebSocketHost",webSocketIP);
-	GetConfigValue<string>("WebSocketPort",webSocketURLExt);
+	GetConfigValue<std::string>("WebServiceIP", webip);
```

Comment:
**NIT** Just an FYI. This works fine but there is an overloaded version of this method you can directly pass the mutex to, which avoids having to created the scoped `std::lock_guard`. See https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/4847aa7fdd30bf31f013cfe74fc3920d302fffe6/src/v2i-hub/SpatPlugin/src/SpatPlugin.cpp#L44C1-L45C1.